plugin/root/README.md

```diff
@@ -2,14 +2,18 @@
 
 ## Name
 
-*root* - simply specifies the root of where to find (zone) files.
+*root* - simply specifies the root of where to find files. 
 
 ## Description
 
 The default root is the current working directory of CoreDNS. The *root* plugin allows you to change
-this. A relative root path is relative to the current working directory.
+this. A relative root path is relative to the current working directory. 
+The *root* directory is currently supported by the following plugins:
```

Comment:
```suggestion

**NOTE: The *root* directory is NOT currently supported by all plugins.** 

Currently the following plugins respect the *root* plugin configuration:
```

Im thinking we should add a Bugs section (like some other plugins), and add this note there as well.

Reply:
Adjusted it, I have also added the "Bugs" section.